packages/react-components/src/components/PictureInPictureInPicture/PictureInPictureInPicture.tsx

```diff
@@ -87,6 +87,7 @@ const handleKeyDown = (e: React.KeyboardEvent<HTMLElement>, onClickCallback: ()
 
 const tileContainerStyles: React.CSSProperties = {
```

Comment:
do we have any guidelines on what types these defined styles should be? There seems to be a lot of choices depending on what kind of fluent component you are dealing with

Reply:
In short: no. But as long as it's strongly typed we don't have a strong bias against one method or another